walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/EventTime.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.lang.Comparable;
+
+/** Class that defines nodes for the start and end times of Events */
+public final class EventTime implements Comparable<EventTime> {
```

Comment:
Is this your writing or something you pulled in?

Reply:
I wrote that class, is that what you're asking?